pyfr/plugins/tavg.py

```diff
@@ -24,6 +26,11 @@ def __init__(self, intg, cfgsect, suffix=None):
         self.mode = self.cfg.get(cfgsect, 'mode', 'windowed')
         if self.mode not in {'continuous', 'windowed'}:
             raise ValueError('Invalid averaging mode')
+        
+        # Std deviation mode
+        self.dev_mode = self.cfg.get(cfgsect, 'std-dev', 'summarise')
```

Comment:
I'd rename this to std_mode.  We may also want to rename the key to something like `std-mode`.  `summary` may be a better name, but am open to suggestions.